mkdocs.yml

```diff
@@ -72,32 +72,46 @@ plugins:
             - /docs/*
 nav:
   - 'Overview': index.md
-  - 'Getting Started': getting-started.md
+  - 'Getting Started': 
+    - 'Single-Cluster': getting-started-single.md
+    - 'Multi-Cluster': getting-started-mc.md
```

Comment:
It's a silly thing, but since these file names end up making part of the final URLs of the pages, would you mind please renaming them without the abbreviation? E.g., `getting-started-single-cluster.md` and `getting-started-multi-ccluster.md`?

Reply:
`getting-started-single-cluster.md` as well?